### PR TITLE
add example release-0.3 to release-0.8 migration to docs

### PR DESCRIPTION
**Pull request by Kolossi**
A PR for consideration to add docs as promised to @paulfantom  on https://kubernetes.slack.com/archives/CFFDS2Z7F/p1624447458459800

Related to Issue https://github.com/prometheus-operator/kube-prometheus/discussions/1154

### CODE REVIEW

**Review by paulfantom — APPROVED** [review]
lgtm

Thank you so much! :100:

File: docs/migration-example/my.release-0.8.jsonnet [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+        apiVersion: 'rbac.authorization.k8s.io/v1',
+        kind: 'Role',
+        metadata: {
+          name: 'node-exporter-psp',
+          namespace: $.values.common.namespace,
+        },
+        rules: [{
+          apiGroups: ['policy'],
+          resources: ['podsecuritypolicies'],
+          verbs: ['use']
```
Comment (paulfantom) [inline docs/migration-example/my.release-0.8.jsonnet:234]:
```suggestion
          verbs: ['use'],
```

**Comment by paulfantom** [comment]
There seems to be a small formatting issue. Could you run `make fmt` and commit changes?

**Comment by Kolossi** [comment]
@paulfantom : I tried to just click to commit the one suggestion, but there seems to be more.  I'll need to wait now until at least tomorrow when I can get back to my source.

**Pull request merged**